extensions/provisioning/verify.go

```diff
@@ -165,8 +165,10 @@ func VerifyCluster(t *testing.T, client *rancher.Client, clustersConfig *cluster
 	podErrors := pods.StatusPods(client, status.ClusterName)
 	assert.Empty(t, podErrors)
 
-	if clustersConfig.ClusterSSHTests != nil {
-		VerifySSHTests(t, client, cluster, clustersConfig.ClusterSSHTests, status.ClusterName)
+	if clustersConfig != nil {
```

Comment:
Is this parent conditional necessary? What was the issue with just having this left as is?

Reply:
Since rke2 templates calls verify cluster without a clusterconfig the test would fail at this point since clusterConfig.ClusterClusterSSHTests was attempting to reference a nil object.